docs/integrations/sources/microsoft-entra-id.md

```diff
@@ -0,0 +1,42 @@
+# Microsoft Entra Id
+The Microsoft Entra ID Connector for Airbyte allows seamless integration with Microsoft Entra ID, enabling secure and automated data synchronization of identity and access management information. With this connector, users can efficiently retrieve and manage user, group, and directory data to streamline identity workflows and ensure up-to-date access control within their applications.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `type` | `string` | Type.  |  |
+| `client_id` | `string` | Client ID.  |  |
+| `client_secret` | `string` | Client secret.  |  |
+| `tenant_id` | `string` | Tenant Id.  |  |
+| `application_id_uri` | `string` | Application Id URI.  |  |
+| `user_id` | `string` | User Id.  |  |
+| `ids` | `array` | Ids.  |  |
+| `types` | `array` | Types.  |  |
+
+## Streams
+| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
+|-------------|-------------|------------|---------------------|----------------------|
+| users | id | No pagination | ✅ |  ❌  |
```

Comment:
Please check instructions how to implement pagination in Microsoft https://learn.microsoft.com/en-us/graph/best-practices-concept#pagination

Reply:
ok

Reply:
@marcosmarxm added pagination . check plz .